### ISSUE TITLE
Initialization with access token wont work

### ISSUE THREAD

**Issue by Hamdan85** [OlympiaAI/open_router#2]
This piece of code from documentation:

```ruby
client = OpenRouter::Client.new(
    access_token: "access_token_goes_here",
    request_timeout: 240 # Optional
)
```

returns 

`initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)

**Comment by obie** [comment]
Will be fixed later today, sorry.

**Comment by obie** [comment]
Fixed in v0.2.1. Thanks for reporting.
